edison/config.py

```diff
@@ -0,0 +1,28 @@
+import secrets
+import importlib, inspect
+import sys, inspect
+
+config_dict = {}
+
+# If config_dict is empty this function builds it dynamically 
```

Comment:
Given that the `get_config_object` function is AFAIK only called once when the Flesk process starts, do you really need to cache the classes in the `config_dict`? It seems to me you can simplify this code.

Also, please use docstrings to document functions and not comments.

I think the docs for this function need to be improved, its doing non-trivial things like use the `inspect` module and returning classes. The motivations and techniques for doing this need to be explained. Try to see of your peers that did not write this code can figure out why you wrote it and what it does...

Reply:
ack